packages/perseus/src/article-renderer.tsx

```diff
@@ -167,13 +172,20 @@ class ArticleRenderer
 
         // TODO(alex): Add mobile api functions and pass them down here
         const sections = this._sections().map((section, i) => {
-            const refForSection = `section-${i}`;
+            const refForSection: any = i;
             return (
-                <DependenciesContext.Provider value={this.props.dependencies}>
+                <DependenciesContext.Provider
+                    key={i}
```

Comment:
Do we really need to be creating multiple dependencies context providers? It doesn't seem specific to each of these sections at all. However, there could be something I'm missing.

For now, I added a key to stop react from throwing a noisy error about not having unique key props for our children in a list.

Reply:
No, I don't think we do.  We could move it to just wrap the final `<div />` that we render these sections into!